src/Apps/W1/Subscription Billing/App/Service Objects/Tables/SubscriptionHeader.Table.al

```diff
@@ -420,9 +420,11 @@ table 8057 "Subscription Header"
                 ServiceCommitment: Record "Subscription Line";
                 ContractsItemManagement: Codeunit "Sub. Contracts Item Management";
             begin
-                if "Source No." <> xRec."Source No." then
+                if "Source No." <> xRec."Source No." then begin
```

Comment:
Since your changing something variant code related.
Shouldn't the variant code get cleared when the `field(50; Type; Enum "Service Object Type")` is changed, too?

Reply:
Good observation. I need to address that.